packages/transport/src/modules/relationships/RelationshipsController.ts

```diff
@@ -150,26 +150,25 @@ export class RelationshipsController extends TransportController {
         return await CoreCrypto.verify(content, signature, relationship.peer.publicKey);
     }
 
-    public async sendRelationship(parameters: ISendRelationshipParameters): Promise<Relationship> {
-        parameters = SendRelationshipParameters.from(parameters);
-        const template = (parameters as SendRelationshipParameters).template;
-        if (!template.cache) {
-            throw this.newCacheEmptyError(RelationshipTemplate, template.id.toString());
-        }
+    public async sendRelationship(parameters: ISendRelationshipParameters): Promise<Result<Relationship>> {
```

Comment:
why have you decided to use a restult as the return value? We don't do this anywhere else. IMO you can just throw in controller functions.

Reply:
The CreateRelationshipUseCase needs to do stuff, after error.transport.relationships.expiredRelationshipTemplate is returned. If I just throw an error, I cannot handle it appropriately anymore.

Reply:
hmm... try..catch?

Reply:
I thought we should avoid try... catch, which is why I decided to use the Result instead of try... catch. But of course I can change it, if you want me to. 😊

Reply:
@jkoenig134 In some controllers we return ValidationResult. However, they have this `items` property which doesn't fit here. Therefore, from my point of view it's okay to return a Result here.

Reply:
I don't like the result because there is no obvious reason to sometimes return a Result and sometimes throw an error .